<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>phase_dialog</class>
 <widget class="QDialog" name="phase_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>相位编辑</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>20</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>相位号：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Phase">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>620</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_basicTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>571</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>基础时间</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最小绿:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>单位延长绿:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>机动黄灯:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>最大绿1:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值限制:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>全红时间:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>最大绿2:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>动态步长:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>红灯保护:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_MinimumGreen">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>25</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_Passage">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>65</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_YellowChange">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>105</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_Maximum1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>25</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_DynamicMaxLimit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>65</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_RedClear">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>105</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_DynamicMaxStep">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>65</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_RedRevert">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>145</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_Maximum2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>25</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>机动清空:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_VehicleClear">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>105</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_flash">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>571</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>密度时间</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>增加初始值:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>递减前时间:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>单位递减率:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>最小间隔:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>递减前车辆:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>递减时间:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>最大初始值:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_AddedInitialCalc">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>增加初始值计算</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_ReduceBy">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>105</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_AddedInitial">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>25</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_TimeBeforeReduction">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>65</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_MinimumGap">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>105</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_CarsBeforeReduction">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>65</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_TimeToReduce">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>65</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_MaximumInitial">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>25</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_dim">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>行人</string>
   </property>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>行人放行:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_PedestrianClear">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>65</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_Walk">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>25</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>行人清空:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_KeepPedRelease">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>保持行人放行</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>620</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_countdown">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>220</y>
     <width>201</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>选项</string>
   </property>
   <widget class="QCheckBox" name="checkBox_AllowDetectorToShieldRequest">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>允许检测器屏蔽请求</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GuaranteeFlowDensityExtendGreen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>保证流量密度延长绿</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DoubleEntryPhase">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>双入口相位</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ConditionService">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>有条件服务有效</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_SimultaneousVoidFail">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>同时空当失效</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(单位：秒)</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>781</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>综合</string>
   </property>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>非感应:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>机动车自动请求:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Startup">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>未启用</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>绿灯</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>机动车绿灯</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>黄灯</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>红灯</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>关灯</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_VehicleRequest">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>未定义</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>最小请求</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>最大请求</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>软请求</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_NoneInduction">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>未定义</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TS非感应1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TS非感应2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TS非感应1-2</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox_AutoFlashIn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>自动闪光进入</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_AutoFlashOut">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>自动闪光退出</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_PedAutoRequest">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>行人自动请求</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_Ring">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>属于环：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Ring">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>781</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>并发相位</string>
   </property>
   <widget class="QPushButton" name="pushButton_phaseConcurrency">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>…</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phaseConcurrency">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>661</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
